jarviscli/plugins/gmail.py

```diff
@@ -1,47 +1,163 @@
+# GMAIL ONLY
```

Comment:
Guess there isn't even a reason this is Gmail only? Added to my list: Make gmail -> mail ;).

Reply:
Yeah, added in case someone tries to send an email through something else...